### ISSUE TITLE
Users with permission for "Create forms", but not "Manage form settings" cannot duplicate forms.

### ISSUE THREAD

**Issue by Kashkin** [verbb/formie#665]
**Description**
A user with "Create forms" and "Manage form" permissions can use the "Save as new form" dropdown option. If they do not have the "Manage form settings" permission, they will receive an error: `"Handle cannot be blank."`

Presumably this is because the handle is not present on the page for the duplication function.

**Additional info**
- Plugin version: 1.4.27
- Craft version: 3.7.10
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.27",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.28](https://github.com/verbb/formie/releases/tag/1.4.28)
